requre/objects.py

```diff
@@ -54,7 +54,7 @@ def __init__(
         self.store_keys = store_keys
 
     @staticmethod
-    def __get_base_keys(func: Callable) -> List[Any]:
+    def _get_base_keys(func: Callable) -> List[Any]:
```

Comment:
What about `get_base_keys` since it is used outside of the class?

Reply:
yep, could be.